web/styles/app_variables.css

```diff
@@ -261,6 +261,7 @@
         231deg 100% 90% / 50%
     );
     --color-compose-chevron-arrow: hsl(0deg 0% 58%);
+    --color-composebox-button-background: hsl(0deg 0% 0% / 5%);
```

Comment:
Let's add `-hover` to the end of these button-background variables, so that a) it's clear those are shown on hover and b) to future-proof against a design that has a background color outside the hover state (heck, it might be worth doing that now, even if you use the `transparent` value).

Reply:
I myself was on the fence about this same decision... Have added a variable for transparent background too now!